### DISCUSSION TITLE
Best practices: where to store user manuals ?

### DISCUSSION THREAD

**Discussion by ech1965** [netbox-community/netbox#8568] [Q&A]
Dear,

I'd like to attach the device user's manual or documentation to a device type ( eg: Switch, Mainboard manual, ...)
It appear it's not possible.
It also appears to be on purpose.

An Url to the manual in Manufacturer site is not acceptable ( rebranding, web site revamp...)
For business reasons, I need the manual to be available "off-line" without requiring internet access and for at least 10-15 years.

What is your advice ?
Thanks in advance
EC

**Comment by candlerb** [comment]
In Netbox you can attach images, but not arbitrary documents like PDFs.

Probably your best bet is to store them in a separate webserver, and in Netbox use a URL Custom Field or a Custom Link to link to them (e.g. from the Device Type).

Another option is to run a wiki, where you can store attachments, and link to the wiki - then you can use it for editable documentation too, more than you can fit in the "comments" field in Netbox.

**Comment by bogdancordos** [comment]
It would be a very handy feature for NetBox to leverage MkDocs and incorporate it as a documentation plugin.
